supervision/classification/core.py

```diff
@@ -65,6 +70,37 @@ def from_yolov8(cls, yolov8_results) -> Classifications:
         confidence = yolov8_results.probs.data.cpu().numpy()
         return cls(class_id=np.arange(confidence.shape[0]), confidence=confidence)
 
+    @classmethod
+    def from_ultralytics(cls, ultralytics_results) -> Classifications:
+        """
+        Creates a Classifications instance from a (https://github.com/ultralytics/ultralytics) inference result.
+
+        Args:
+            ultralytics_results (ultralytics.yolo.engine.results.Results): The output Results instance from ultralytics model
+
+        Returns:
+            Classifications: A new Classifications object.
+
+        Example:
+            ```python
+            >>> import cv2
+            >>> from ultralytics import YOLO, FastSAM, SAM, RTDETR
+            >>> import supervision as sv
+
+            >>> image = cv2.imread(SOURCE_IMAGE_PATH)
+            >>> model = YOLO('yolov8s.pt')
```

Comment:
Let's update the example to use only detection and segmentation models here.

Reply:
That is my point exactly. We are documenting `sv.Classifications.from_ultralytics` but we are showing examples of usage with detection models. We should show classification model examples.

Reply:
Done! Although I can see only yolo models for classification on ultralytics https://docs.ultralytics.com/tasks/classify/